### PR TITLE
Updated WinRT build to include newly added files

### PR DESCRIPTION
**Pull request by dotMorten**
Updated WinRT build to include newly added files, and allowed more serial devices to be discovered.

These changes are cherry-picked from @donald-hanson's PR #1045, but excludes all the parts that adds a new WinRT Dynamic Library.

If you prefer splitting the .NET and UWP Builds out in a separate repo, this would be the minimum required changes to start that work, and brings the WinRT static library build in line with the Windows build.

All credits goes to @donald-hanson, as he did all the hard work. However he said he's been pretty busy lately and OK with me taking over his (great) UWP work.

As a prrof of concept, I've also created a separate repo with the UWP Dynamic DLL wrapper bits removed from the other PR + also a sample app, and then added OpenZWave as a submodule instead.

### CODE REVIEW

**Comment by dotMorten** [comment]
Also fixed a bug occurring after a driver has been removed (attempt to read/write to the port after the driver has been removed)

File: cpp/src/platform/winRT/SerialControllerImpl.cpp [hunk 1]
```diff
@@ -189,6 +189,8 @@ void SerialControllerImpl::StartReadTask()
 				create_task(m_serialDevice->InputStream->ReadAsync(buffer, readBufferLength, InputStreamOptions::None))
 					.then([&, this](IBuffer ^ outBuffer)
 				{
+					if (token.is_canceled()) 
+						return;
```
Comment (dotMorten) [inline cpp/src/platform/winRT/SerialControllerImpl.cpp]: This fixes a crash that occurs after a driver is removed.

**Comment by dotMorten** [comment]
Ping

**Comment by Fishwaldo** [comment]
Sorry for the slow response. Real life getting in the way. This looks good.

**Comment by dotMorten** [comment]
No worries. Life is important 😁

**Pull request merged**